src/components/learner-credit-management/BudgetDetailAssignments.jsx

```diff
@@ -26,20 +29,26 @@ const BudgetDetailAssignments = ({
     assignmentConfigurationUUID,
     enterpriseId,
   });
-
+  
   if (!isTopDownAssignmentEnabled || !isAssignableBudget) {
     return null;
   }
-
+  
   if (!hasContentAssignments && hasSpentTransactions) {
     return (
       <AssignMoreCoursesEmptyStateMinimal />
     );
   }
-
+  
+  useEffect(() => {
+    if (locationState?.budgetActivityScrollToKey === 'assigned') {
+      assignedHeadingRef.current?.scrollIntoView({ behavior: 'smooth' });
```

Comment:
I believe we may need to `delete` the locationState after we've handled it here. Otherwise, for example, after I scroll to the "Assigned" / "Spent" tables, and then do a full page refresh, the `locationState` is unchanged and incorrectly scrolls me down to the "Assigned" / "Spent" tables (depending on which scroll-to link was last clicked).